src/main/java/net/sf/jabref/logic/xmp/XMPUtil.java

```diff
@@ -626,11 +626,10 @@ private static void writeToDCSchema(XMPSchemaDublinCore dcSchema,
              */
             if ("author".equals(field)) {
                 String authors = resolvedEntry.getField(field);
-                AuthorList list = AuthorList.getAuthorList(authors);
+                AuthorList list = AuthorList.getAuthors(authors);
 
-                int n = list.size();
-                for (int i = 0; i < n; i++) {
-                    dcSchema.addCreator(list.getAuthor(i).getFirstLast(false));
```

Comment:
So the one-liner to get a List <Auhtor> is

AuthorList.getAuthors(String).getAuthors()?

Maybe not a huge difference but I sort of liked the previous naming better...